public/script.js

```diff
@@ -1,5 +1,44 @@
-const earthDateParagraph = document.querySelector("#earth-date");
 const earthDate = new Date();
-const dateText = document.createTextNode(earthDate.toLocaleString());
 
-earthDateParagraph.appendChild(dateText);
+function displayCurrentEarthDate() {
+  const earthDateParagraph = document.querySelector("#earth-date");
+  const dateText = document.createTextNode(earthDate.toLocaleString());
+
+  earthDateParagraph.appendChild(dateText);
+}
+
+displayCurrentEarthDate();
+
+function displayKennedySpaceCenterTime() {
+  const kennedyTimeParagraph = document.querySelector("#kennedy-time");
+  const kennedyTime = earthDate.toLocaleTimeString(undefined, {
+    timeZone: "America/New_York",
+  });
+  const kennedyTimeText = document.createTextNode(kennedyTime);
+
+  kennedyTimeParagraph.appendChild(kennedyTimeText);
+}
+
+displayKennedySpaceCenterTime();
+
+function displayOffsetTimeToKennedySpaceCenter() {
+  const kennedyOffsetTime = document.querySelector("#kennedy-offset");
+  const utcOffsetTimeInHours = earthDate.getTimezoneOffset() / 60;
+  const timeBetweenKennedyAndUtc = utcOffsetTimeInHours + 5;
+  const kennedyOffset = document.createTextNode(timeBetweenKennedyAndUtc);
+  kennedyOffsetTime.appendChild(kennedyOffset);
+  const aheadOfKennedy = document.createTextNode(
+    " hours ahead of Kennedy Space Center)"
+  );
+  const behindKennedy = document.createTextNode(
+    " hours behind of Kennedy Space Center)"
+  );
+
+  if (timeBetweenKennedyAndUtc > 0) {
+    kennedyOffsetTime.appendChild(aheadOfKennedy);
+  } else {
+    kennedyOffsetTime.appendChild(behindKennedy);
+  }
```

Comment:
Could use string interpolation to build the string, and then put it all in as a child text node in one go?

Reply:
I think you could combine both ahead and behind to be just the one string!

Reply:
Something like `${time} hours ${isAhead ? "ahead" : "behind"} of KSC`

(...but done properly)

Reply:
Sounds good, will take a look at that now